src/kixi/datastore/schemaextracter.clj

```diff
@@ -17,8 +19,7 @@
 (defn metadata->file
   [filestore metadata]
   (let [id (get-in metadata [:id])
-        f (File/createTempFile id ".tmp")]
-    (.deleteOnExit f)
+        f (temp-file id ".tmp")]
```

Comment:
Are you passing in two params when the temp-file function only accepts the name?